src/freenas/etc/subgid

```diff
@@ -0,0 +1 @@
+0:1000000:1000000000
```

Comment:
Great question, I would guess minimal since as I understand this is for child namespaces and it is for id mapping on to containers processes and back to the host. But definitely needs more thought as I am not an expert in this. This was the bare minimal I had to do to get privileged containers working.

Reply:
Ok, then we should probably dynamically generate this and perhaps have it be configurable?

Reply:
https://github.com/truenas/middleware/blob/4bd7d7dd138cdb44dab2c856a7c7141a352286d2/src/middlewared/middlewared/plugins/idmap_/idmap_constants.py#L9

You can maybe reserve a range above `TRUENAS_IDMAP_MAX`. I originally sliced it off for clustered users and groups, but you can in theory use it here.

Reply:
That said, maybe slice off no more than you need.